C/cargo_license/build_tarballs.jl

```diff
@@ -0,0 +1,45 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "cargo_license"
+version = v"0.4.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/onur/cargo-license.git", "f489e18b0fc45ab31d9766841e7f551e96ee6a8c"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+
+
+cd $WORKSPACE/srcdir/cargo-license
+
+cargo build --release --all
+
+mkdir ${bindir}
+cp target/${rust_target}/release/cargo-license${exeext} ${bindir}/
+
```

Comment:
Why so many (and consecutive) blank lines?

Reply:
Fixed. :)